### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by SumaiaSorna**
### Work Done

- Added title with my name
- Attached a recent image of mine with bio and skills
- Designed the nav link, which will take you  to the specific section on the page
- Added  project (Refactor Code) with title
- "Refactor Code" project is clickable
- The other six projects cards are in a flex
- Responsive Layout (nav bar, project cards, about me section)
- Added comments in the HTML and CSS files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            <p>
+              Refactoring Horiseon-marketing-website with semantic html for
+              improved accessibility.
+            </p>
+          </div>
+        </div>
+        <!-- six projects in a flex container-->
+        <div class="cards-container">
+          <div class="small-project-card">
+            href="">
```
Comment (chelseanicholls) [inline index.html]: This line is showing as text on your portfolio, please delete

File: index.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+              src="./assets/images/project-7.jpg"
+              alt="placeholder project image"
+            />
+            <h3>Project 7</h3>
+            <p>coming soon</p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <!-- footer section  and added id for the contact me section -->
```
Comment (chelseanicholls) [inline index.html:142]: Ensure there are spaces between each section to maximise readability

File: index.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    <footer id="Contact-me">
+      <h3>Sum<span>aia</span> Salah<span>uddin</span></h3>
+      <div class="contact-me">
+        <a href="https://www.linkedin.com/in/sumaia-salahuddin-519521218/"
```
Comment (chelseanicholls) [inline index.html]: Possibly add target="_blank" to these links so they open in a new tab

File: index.html (hunk 1 above)
Comment (SumaiaSorna) [reply index.html]: Thank you chelsea

File: index.html (hunk 3 above)
Comment (SumaiaSorna) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (SumaiaSorna) [reply index.html:142]: Done

**Pull request merged**